### PR TITLE
Network Plugins: Add missing plugin to the summary section

### PR DESCRIPTION
**Pull request by baczus**
PR aligns info in [kubenet](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#kubenet) section with the summary.

### CODE REVIEW

**Comment by bradtopol** [comment]
/assign
Hi @cmluciano can you do a technical review of this?  Simply add a lgtm if it is technically correct? Thanks!
/cc @cmluciano

**Review by cmluciano — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ This option is provided to the network-plugin; currently **only kubenet supports
 ## Usage Summary
 
 * `--network-plugin=cni` specifies that we use the `cni` network plugin with actual CNI plugin binaries located in `--cni-bin-dir` (default `/opt/cni/bin`) and CNI plugin configuration located in `--cni-conf-dir` (default `/etc/cni/net.d`).
-* `--network-plugin=kubenet` specifies that we use the `kubenet` network plugin with CNI `bridge` and `host-local` plugins placed in `/opt/cni/bin` or `cni-bin-dir`.
+* `--network-plugin=kubenet` specifies that we use the `kubenet` network plugin with CNI `bridge`, `lo` and `host-local` plugins placed in `/opt/cni/bin` or `cni-bin-dir`.
```
Comment (bradtopol) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:162]: Great job!

**Comment by bradtopol** [comment]
Well done!
/lgtm
/approve

**Pull request merged**